README.md

```diff
@@ -1,7 +1,17 @@
-```yaml
+Installation instructions:
+
+1. Install HACS for Home Assistant
+2. Go to Community-Store-Grocy
+3. Install Grocy
+4. Restart Home Assistant
+5. Go to Grocy-Wrench icon-Manage API keys-Add
+6. Copy resulting API key and Grocy URL and input this in configuration.yaml:
+
 grocy:
```

Comment:
could you readd the formatting for better readability?

Reply:
Done, hope it's correct now, this is my first time with both the GitHub text editor and the pull request system :)